core/src/test/java/org/pentaho/di/connections/vfs/provider/BaseVFSConnectionProviderTest.java

```diff
@@ -24,62 +24,5 @@ public void setup() {
     baseVFSConnectionDetails = Mockito.spy( BaseVFSConnectionDetails.class );
   }
 
-  @Test
-  public void testWithProvidedRootPath() throws KettleException {
```

Comment:
All of these were moved/refactored into `DefaultVFSConnectionManagerWrapperTest`.